### PR TITLE
Add mina deploy

### PR DESCRIPTION
**Pull request by enricostano**
### WAT

Using [Mina deploy](http://nadarei.co/mina/) to manage deploys, same as backend project.

### CODE REVIEW

File: config/deploy.rb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  deploy do
+    invoke :'git:clone'
+    invoke :'deploy:link_shared_paths'
+    queue 'npm install'
+    queue 'npm run build'
+    invoke :'deploy:cleanup'
+
+    to :launch do
+      queue "mkdir -p #{deploy_to}/#{current_path}/tmp/"
+      queue "touch #{deploy_to}/#{current_path}/tmp/restart.txt"
```
Comment (andresgutgon) [inline config/deploy.rb]: This smell like rails :)

File: config/deploy.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      # TODO: Find a way to run this in background
+      # TODO: Restart?
+      queue 'npm run start &'
```
Comment (andresgutgon) [inline config/deploy.rb]: I think `start` command is doing it for you:

``` json
"start": "concurrent --kill-others \"npm run start-prod\"",
```

`--kill-others`

https://github.com/coopdevs/katuma-web/blob/develop/package.json#L21

File: config/deploy.rb (hunk 1 above)
Comment (enricostano) [inline config/deploy.rb]: ooops :grimacing:

**Comment by enricostano** [comment]
ready to be merged! :smiley:

File: .ruby-version [hunk 3]
```diff
@@ -0,0 +1 @@
+2.1.4
```
Comment (andresgutgon) [inline .ruby-version:1]: Arggg! ruby :trollface:

**Comment by andresgutgon** [comment]
:+1:

File: .ruby-version (hunk 3 above)
Comment (enricostano) [inline .ruby-version:1]: ya.... :grin:

**Pull request merged**